mpas_analysis/shared/mpas_xarray/mpas_xarray.py

```diff
@@ -543,4 +532,59 @@ def _parse_dataset_time(ds, inTimeVariableName, calendar,
     return dsOut  # }}}
 
 
+def process_chunking(ds, chunking):  # {{{
+    """
+    Computes chunking for a dataset.
+
+    Parameters
+    ----------
+    ds : input xarray Dataset.
```

Comment:
Use double back-quotes in the same way you would use single back-quotes in Markdown.